logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -338,8 +350,73 @@ actual class CallManagerImpl(
         return AvsCallBackError.NONE.value
     }
 
+    /**
+     * onCallingReady
+     * Will start the handlers for: ParticipantsChanged, NetworkQuality, ClientsRequest and ActiveSpeaker
+     */
+    private fun onCallingReady() {
```

Comment:
I would suggest to extract it to a separate file, CallManager contains is becoming bigger and bigger

Reply:
I think it will be better if we move this after we have your PR merged with the scenarios